packages/Switch/stories/Switch.stories.js

```diff
@@ -28,37 +28,5 @@ storiesOf(storyName, module)
         size={select("size", [types.SMALL, types.MEDIUM, types.LARGE], "medium")}
       />
     </Story>
-  ));
-
-storiesOf(`${storyName}/Examples`, module)
-  .add("Switch sizes", () => (
-    <Story>
-      <h4>
-        <code>size=&quot;small&quot;</code>
-      </h4>
-      <Example size={Switch.types.size.SMALL} />
-      <p />
-      <h4>
-        <code>size=&quot;medium&quot;</code>
-      </h4>
-      <Example size={Switch.types.size.MEDIUM} />
-      <p />
-      <h4>
-        <code>size=&quot;large&quot;</code>
-      </h4>
-      <Example size={Switch.types.size.LARGE} />
-    </Story>
   ))
-  .add("Disabled switch", () => (
-    <Story>
-      <h4>
-        <code>isDisabled</code>
-      </h4>
-      <Example isDisabled isChecked={false} />
-      <p />
-      <h4>
-        <code>isDisabled isChecked</code>
-      </h4>
-      <Example isDisabled isChecked />
-    </Story>
-  ));
+  .add("Variations", () => <Variations />);
```

Comment:
This is good for this PR, but I think it's worth mentioning this "storiesOf" style is old, and ideally we'd use the new "Compact Stories Format" (CSF).  Here's an example:
https://github.com/acl-services/paprika/blob/master/packages/Heading/stories/Heading.stories.js#L26-L34

With this old "storiesOf" style we can't order the stories or the components the way we'd like.  

But yeah, this task wasn't about refactoring the documentation, adding this Variations story is a nice bonus though 👍